.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
```

Comment:
While this is false, users will only be able to select a form. This is why a "Question" form is necessary imho